src/core/viz/expressions/aggregation/clusterAggregation.js

```diff
@@ -7,8 +7,10 @@ import { checkInstance, checkType } from '../utils';
  * Aggregate using the average. This operation disables the access to the property
  * except within other cluster aggregate functions.
  *
- * @param {carto.expressions.prop} property - Column of the table to be aggregated, must be a date or a number
- * @return {carto.expressions.prop} Aggregated column
+ * Note: `clusterAvg` input property can only be a {@link carto.expressions.prop|carto.expressions.prop}, not other expressions.
```

Comment:
can only be an expression created by {@link carto.expressions.prop|carto.expressions.prop} ?

Reply:
`globalAvg` can only be created by {@link carto.expressions.prop|carto.expressions.prop}, not other expressions. ?

Reply:
Yes :+1: `clusterAvg` in this case

Reply:
(Ups, I deleted my old message)

 > `clusterAvg` can only be created by {@link carto.expressions.prop|carto.expressions.prop}, not other expressions.